flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -7,7 +7,6 @@ The Apache Software Foundation (http://www.apache.org/).
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
 - org.apache.parquet:parquet-hadoop:1.11.1
-- org.apache.parquet:parquet-format:2.4.0
```

Comment:
there's a specific include in the hive pom. Are we publishing hive connectors for different versions? (god I hope not)
maybe this entry is correct for _some_ hive version.

Reply:
Well, we are deploying all those flink-sql-connector-hive-{1.22,2.2.0,2.3.6,3.1.2} versions. But they have their own NOTICE files.
The shade plugin output for `flink-connector-hive` doesn't include this dependency. We are not shading `flink-connector-hive` multiple times for different versions.

Reply:
oki. let's remove it then, but maybe ping one of the hive guys to double check that the packaging is correct.

Reply:
@JingsongLi Can you double check that it is fine to remove `org.apache.parquet:parquet-format:2.4.0` from the NOTICE file. @tillrohrmann added it in a hotfix in July: https://github.com/apache/flink/commit/4df425bde344aee8f5a0208df8f5377342af25ff.
In the current master, the dependency doesn't show up anymore in the maven shade output of the module.

Reply:
This is because hive bundle the flink-parquet.
The parquet version of previous flink-parquet required this `parquet-format:2.4.0`.
But after https://issues.apache.org/jira/browse/FLINK-19137 , the new parquet version don't need this dependency.

We can just remove it, I think this is a mistake of https://github.com/apache/flink/commit/fb4ebbb1941c37fd1176e31052529f6cdd92faff

Reply:
Thanks a lot for confirming. I just searched the entire Flink build output, and `parquet-format` is not mentioned there at all.
I'll remove this entry from the NOTICE file then.